radio/src/translations/de.h.txt

```diff
@@ -1128,10 +1128,10 @@
 #define TR_SCRIPT              "Lua-Skript"
 #define TR_INPUTS              "Eingaben"
 #define TR_OUTPUTS             "Ausgaben"
-#define TR_EEBACKUP            "\002[ENTER Long] Backup EEPROM->SD-Karte"
-#define TR_FACTORYRESET        "\004[MENU Long] ALLES komplett löschen"
-#define TR_CONFIRMRESET        "ALLE Modelle + Einst. löschen?"
-#define TR_TO_MANY_LUA_SCRIPTS "Zu viele Lua-Skripte!!"
+#define TR_EEBACKUP            TR("ENTER lang  EEPROM->SD"," [ENTER Long] Backup EEPROM->SD-Karte")
+#define TR_FACTORYRESET        TR("Menü lang  Reset alles","  [MENU Long] Komplett reset")
```

Comment:
I mean the \002 and \004 chars. Also I think Menü means Speiskarte while here Menu is appropiate, also since the physical key is labelled MENU. 

>>> len("[MENU lang] Werksreset")
22